### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Tamas Kadar
```
Comment (godrei) [inline LICENSE]: use this licence file: https://github.com/bitrise-steplib/step-template/blob/master/LICENSE

File: README.md [hunk 2]
```diff
@@ -1 +1,93 @@
-# steps-bitrise-run
\ No newline at end of file
+# Bitrise Run
+
+Lets you run a bitrise workflow from your git repo's bitrise.xml
```
Comment (godrei) [inline README.md]: Runs a specified workflow from the given bitrise.yml

**Comment by KTamas-Bitrise** [comment]
@godrei done

File: step.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -ex
+echo "Workflow id: ${workflow_id}"
+echo "Bitrise config path: ${bitrise_config_path}"
```
Comment (godrei) [inline step.sh]: please validate if `workflow_id` is empty
please validate if `bitrise_config_path ` is exist

File: step.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
+#
```
Comment (godrei) [inline step.yml]: please remove every comment

File: step.yml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+# - Bitrise.yml format spec: https://github.com/bitrise-io/bitrise/blob/master/_docs/bitrise-yml-format-spec.md
+# - Bitrise docs: http://devcenter.bitrise.io/
+# - Bitrise CLI guides: http://devcenter.bitrise.io/bitrise-cli/
+
+title: |-
+  Bitrise Run
+summary: |
+  Lets you run a bitrise workflow from your git repo's bitrise.xml
+description: |
+  Lets you run a bitrise workflow from your git repo's bitrise.xml
```
Comment (godrei) [inline step.yml]: Runs a specified workflow from the given bitrise.yml

File: step.yml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+support_url: https://github.com/bitrise-steplib/step-bitrise-run/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+
+# Type tags are used for categorizing steps, for easier step discovery in Step Libraries.
+# You can find more information about type tags in the Step Development Guideline:
+# https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md
+type_tags:
+  - deploy
```
Comment (godrei) [inline step.yml]: utility

File: step.yml [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+      title: "Workflow ID"
+      summary: ID (name) of the Bitrise workflow to run.
+      description: |
+        ID (name) of the Bitrise workflow to run.
+      is_expand: true
+      is_required: true
+      value_options: []
+  - bitrise_config_path: "./bitrise.yml"
+    opts:
+      title: "bitrise.yml file"
```
Comment (godrei) [inline step.yml]: bitrise.yml path

File: step.yml [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+      summary: The location of the bitrise.yml file in your git repo.
```
Comment (godrei) [inline step.yml]: The Bitrise configuration (bitrise.yml) file path.

File: step.yml [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+      description: |
+        The location of the bitrise.yml file in your git repo.
```
Comment (godrei) [inline step.yml]: The Bitrise configuration (bitrise.yml) file path.

File: step.yml [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+      is_expand: true
+      is_required: false
```
Comment (godrei) [inline step.yml]: is_required: true

File: step.sh (hunk 3 above)
Comment (KTamas-Bitrise) [reply step.sh]: done

File: step.yml (hunk 4 above)
Comment (KTamas-Bitrise) [reply step.yml]: done

File: step.yml (hunk 6 above)
Comment (KTamas-Bitrise) [reply step.yml]: done

File: step.yml (hunk 7 above)
Comment (KTamas-Bitrise) [reply step.yml]: done

File: step.yml (hunk 8 above)
Comment (KTamas-Bitrise) [reply step.yml]: done

File: step.yml (hunk 9 above)
Comment (KTamas-Bitrise) [reply step.yml]: done

File: step.yml (hunk 10 above)
Comment (KTamas-Bitrise) [reply step.yml]: done

File: step.yml [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+      value_options: []
+
+outputs:
```
[the remark this answers is not in the document]
Comment (KTamas-Bitrise) [reply step.yml]: done

**Pull request merged**